plugins/backend/ttrss/ttrssAPI.vala

```diff
@@ -191,7 +191,15 @@ public class FeedReader.ttrssAPI : GLib.Object {
 		if(error == ConnectionError.SUCCESS)
 		{
 			var response = message.get_response_object();
-			unread = int.parse(response.get_string_member("unread"));
+			int64 parsed = 0;
+			bool res = int64.try_parse(response.get_string_member("unread"), out parsed);
+			if(res) {
+				unread = (int) parsed;
+			}
+			else
+			{
+				Logger.warning("Could not parse unread articles");
```

Comment:
I wonder if this should be `error` level?

Reply:
I went with the web dev mantra of, "Should this wake up a sysadmin at night?" Yes: `error`, No: `warning`. Didn't seem severe enough that it warranted an error log level...